src/client/utils/StorageContainer.test.tsx

```diff
@@ -1,27 +1,29 @@
-import { mount } from 'enzyme'
 import React from 'react'
+import userEvent from '@testing-library/user-event'
```

Comment:
Since we export all things `'@testing-library/react'` from the `/test/utils.tsx` file shouldn't `userEvent` rather be imported from there instead? Or have I completely misunderstood? 🙈

Reply:
Hmm should we? It comes from a different package though. Not `@testing-library/react` I mean.

Like I wrote above, I don't love that convention but it's what is recommended in the official guide :P

Reply:
Oh, sorry I just read it as `'@testing-library/react'`, never mind! 😳 Or should we export everything related to testing packages from that `utils` file just to be consistent? Or what's least confusing?

Reply:
This is most explicit so that's why it wins for me 🙃

Reply:
It's good, let's keep it like this! 👍